uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-calendar.html

```diff
@@ -0,0 +1,386 @@
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--type--sickness"><span>AM</span></div>
+            <span class="chr_leave-calendar__legend__label">Sick AM Only</span>
+          </div>
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--type--sickness"><span>PM</span></div>
+            <span class="chr_leave-calendar__legend__label">Sick PM Only</span>
+          </div>
+        </div>
+        <div class="col-sm-3">
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--requested chr_leave-calendar__item--type--sickness"></div>
+            <span class="chr_leave-calendar__legend__label">Requested Sick</span>
+          </div>
+        </div>
+        <div class="col-sm-3">
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--requested chr_leave-calendar__item--type--sickness"><span>AM</span></div>
+            <span class="chr_leave-calendar__legend__label">Requested Sick AM</span>
+          </div>
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--requested chr_leave-calendar__item--type--sickness"><span>PM</span></div>
+            <span class="chr_leave-calendar__legend__label">Requested Sick PM</span>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-3">
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--type--weekend"></div>
+            <span class="chr_leave-calendar__legend__label">Weekend</span>
+          </div>
+        </div>
+        <div class="col-sm-3">
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--type--public-holiday"></div>
+            <span class="chr_leave-calendar__legend__label">Public Holiday</span>
+          </div>
+        </div>
+        <div class="col-sm-3">
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--type--non-working"></div>
+            <span class="chr_leave-calendar__legend__label">Non Working Day</span>
+          </div>
+        </div>
+        <div class="col-sm-3">
+          <div class="chr_leave-calendar__legend__entry">
+            <div class="chr_leave-calendar__item chr_leave-calendar__item--type--late chr_leave-calendar__item--alt"><span>L</span></div>
+            <span class="chr_leave-calendar__legend__label">Late</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+</section>
+<style>
+  /* Absence types (to be created dynamically via AngularJS) */
+  .chr_leave-calendar__item--type--leave {
```

Comment:
Yes, as the comment says, this list will have to be generated dynamically via angular, hence the reason why it's in the template